### ISSUE TITLE
FOSDEM 2016 Open Source Design devroom

### ISSUE THREAD

**Issue by belenbarrospena** [opensourcedesign/events#22]
Just realised today that the deadline for devroom proposals is October 9th. So we better get moving! 

Below is a proposal draft. Please comment / amend as you see fit:

Title of devroom:
Open source design

Elaborate description of proposed devroom (including possible topics):
The first open source design devroom last year was a success, with a healthy number of proposals submitted and great attendance. We would like to repeat this year. The goal of this devroom is to showcase design work done within free and open source projects, and strengthen the connections between designers involved in FOSS. We mean "design" in a broad sense, and we include within that term typography, graphic design, interaction design, user experience and design research activities. 

Possible topics include:
- FOSS design tools
- Design techniques and deliverables
- Design - engineering collaboration approaches
- User engagement and design research activities 
- Project case studies 

After receiving several requests, we will probably dedicate an hour to project pitches, so that projects looking for design contributions can introduce themselves and establish contact with designers eager to help. 

Why does it fit FOSDEM?
FOSS projects bring in particular challenges that affect almost every aspect of design. Techniques and approaches used in closed software settings must be re-purposed and adapted to FOSS. This is hard to do on your own. We hope that sharing experiences across projects and communities will help establish a set of practices that are effective, flexible and transferable across FOSS projects. 

The devroom last year also proved to be a great way of connecting designers with FOSS projects looking for design contributors.

**Comment by plastelina** [comment]
@belenbarrospena , my only concern is that this message is on github, where it won't reach many designers...maybe it's worth to also drop it on few design forums / twitter to actually make designers aware, last year there were... 5 designers in the room (?)...what's the point of pitching, if there is none to pitch too...

I think the topics are great!
I'd love to talk about engineering collaboration approaches, could you point me where do I submit the proposal?

Thanks!

**Comment by ei8fdb** [comment]
If needs be I can do a blog post like last year with pointers to where to go, IRC channel, etc and put it on the twitters to direct people?

http://www.ei8fdb.org/thoughts/2014/10/fosdem-2015-call-for-papers-on-open-source-design/

Bernard

**Comment by plastelina** [comment]
This is awesome job! Thanks!!! :+1:

**Comment by belenbarrospena** [comment]
@plastelina if the devroom proposal is accepted, the call for talks should come out on November 2nd, according to the FOSDEM website

https://fosdem.org/2016/news/2015-09-24-call-for-participation/

That's when you would submit your proposal :)

**Comment by plastelina** [comment]
Yup, read your post @belenbarrospena  ;).

**Comment by evalica** [comment]
Regarding the last year success, should we mention somewhere that a git organization and a website has been put into place? Having the devroom again next year will increase our chances in expanding the community and allowing more developers know that there is a place where they can get design help. 

Yes, excellent point. What about something like this:

<blockquote>The first open source design devroom in FOSDEM 2015 was a success, with a healthy number of proposals submitted and great attendance. It was also the beginning of an Open Source Design community (http://opensourcedesign.net/) with several iniatives, between them a "jobs board" where FOSS projects looking for design contributors can connect with available designers. We would like to repeat next year. </blockquote>

And also, in the "Why does it fit FOSDEM?" section:

<blockquote>Having a devroom again next year will increase our chances to expand the community and will give us the opportunity to spread the word between developers regarding how and where to get design help for their projects.</blockquote>

Also I don't see any mentions about the Design Desk. 

Well, I was thinking of applying for booth space for that, which is a separate process. Now, we could also make the design desk part of the devroom instead, and block a few hours for that instead of having a full day of talks. I wonder what people think would be best.

**Comment by belenbarrospena** [comment]
Right folks: submission for the Open Source Design devroom at FOSDEM 2016 has gone out. I've put myself as the main contact, and Bernard Tyers (@ei8fdb) as the secondary one. If anybody will like to be kept up to date, just send me an email address and I'll keep you posted, although I will keep updating this thread as things happen. Acceptance notifications are supposed to get out on October 19th. 

The content of the proposal was as follows:

Devroom Request for FOSDEM 2016

Title: Open source design
Coordinator: Belen Barros Pena
Coordinator email: belenbarrospena@gmail.com
Secondary contact: Bernard Tyers
Secondary email: b@runningwithbulls.com

Description:
The first Open Source Design devroom in FOSDEM 2015 was a success, with a healthy number of proposals submitted and great attendance. It was also the beginning of an Open Source Design community (http://opensourcedesign.net/) with several initiatives, between them a "jobs board" where FOSS projects looking for design contributors can connect with available designers. We would like to repeat the experience next year.

The goal of this devroom is to showcase design work done within free and open source projects, and strengthen the connections between designers involved in FOSS. We mean "design" in a broad sense, and we include within that term typography, graphic design, interaction design, user experience and design research activities.

Possible topics include:

```
FOSS design tools
Design techniques and deliverables
Design - engineering collaboration approaches
User engagement and design research activities
Project case studies
```

After receiving several requests, we will probably dedicate an hour to project pitches, so that projects looking for design contributions can introduce themselves and establish contact with designers eager to help.

Coordinator's affinity to the topic of the devroom:

I have been the interaction designer of OpenEmbedded (www.openembedded.org) and Yocto Project (www.yoctoproject.org) since 2012, and one of the instigators of the Open Source Design devroom last year.

Why does it fit FOSDEM

FOSS projects bring in particular challenges that affect almost every aspect of design. Techniques and approaches used in closed software settings must be re-purposed and adapted to FOSS. This is hard to do on your own. We hope that sharing experiences across projects and communities will help establish a set of practices that are effective, flexible and transferable across FOSS projects.

The devroom last year also proved to be a great way of connecting designers with FOSS projects looking for design contributors. Having a devroom again next year will increase our chances to expand the community and will give us the opportunity to spread the word between developers regarding how and where to get design help for their projects.

Relevant URLs:
https://github.com/opensourcedesign
http://opensourcedesign.net/

Timeslot:
Full day

Special requirements:
None

**Comment by plastelina** [comment]
This is fantastic! Thank you for putting this together! Ace! :+1:

**Comment by Xaviju** [comment]
Great! Love it! Thanks a lot for the proposal!

**Comment by evalica** [comment]
I looks great! Good job ;)

**Comment by jancborchardt** [comment]
Aaawesome! :) Thanks for leading that effort again @belenbarrospena!

**Comment by belenbarrospena** [comment]
Hi all, 

The call for stands at FOSDEM is also open 
https://submission.fosdem.org/stands.php

Deadline is November 13th. 

A stand would seem to me the most logical way of setting up the "design helpdesk" or "clinic". Unfortunately, I've volunteered to manage the OpenEmbedded stand this year, and the design devroom, so I have my hands full and I don't think I can take anything else.

If anybody would like to step up to organise the design help desk, feel free to send the stand application form. If applying for an "official stand" is a bit too scary, we could run the design desk informally in the cafeteria  one of the days. If that works well, we could think of organising a "formal" one for 2017. 

Any thoughts?

**Comment by ei8fdb** [comment]
Hi all,

I’d be willing to submitting the design stand application if someone else wants to co-organise.

I’ll be helping out in the devroom too so won’t be able to spend all day on the stand as
well.

If not, how about we do an ad-hoc design desk each day for 1/2 a day, morning or afternoon? I could do 1-2 hours each day.

Any takers?

Bernard

---

Bernard Tyers
E: ei8fdb@ei8fdb.org | T: @bernardtyers

Talk to me privately. My public key is here: https://keybase.io/ei8fdb

**Comment by jancborchardt** [comment]
I’m up for assisting with the design stand, but I’ll be helping out in the devroom also and need to be at the ownCloud booth as well.

Who wants to take the helm on the design stand? @evalica @Xaviju @plastelina maybe?

**Comment by Xaviju** [comment]
I'll be happy to help, but I also should probably stay at the [Taiga](https://github.com/taigaio) stand so I think 1-2h each day is fine. If we are not enough, I agree with the @ei8fdb idea of 1/2 day.

**Comment by plastelina** [comment]
Helloh! Sure thing - happy to help. I can take a lead on this. Can we do half day, consultation, pretty please? 
I can mostly help with print, identity and UI design in terms of advise. I can prepare large poster that will say Design Clinic with the current logo and some stickers / pencils ( can get it sorted for free)? I'm free Monday-Tuesday (26-27/10), will login into IRC to chat in more detail. 
@jancborchardt @Xaviju @evalica @belenbarrospena @ei8fdb

**Comment by gillisig** [comment]
I would love to come and help out. But unfortunately I am not sure I can justify the expense to travel there, since it would be coming out of my own pocket :/

**Comment by belenbarrospena** [comment]
Well, we made it! :) Got an email this morning notifying the acceptance of the Open Source Design devroom for FOSDEM 2016. They have scheduled us for Saturday this year.

They will be sending instructions soon. Next step for us should be putting together a call for session proposals.

**Comment by belenbarrospena** [comment]
A couple of more messages:
1. FOSDEM keeps a mailing list for devroom managers. If you want to be subscribed to that mailing list to keep up to date with organisation news, let me know.
2. We have a mailing list for our devroom: open-design-devroom@lists.fosdem.org
   To subscribe, go to 

https://lists.fosdem.org/listinfo/open-design-devroom

**Comment by jancborchardt** [comment]
@belenbarrospena aaaawesome! You can also set me as devroom manager and subscribe me to the first list.

**Comment by belenbarrospena** [comment]
@jancborchardt Will do :) Which email address do you want me to give them?

**Comment by jancborchardt** [comment]
My hey at jancborchardt one. :)

**Comment by GarthDB** [comment]
Congrats!

**Comment by belenbarrospena** [comment]
@jancborchardt Got it. I've asked them to subscribe you.

**Comment by belenbarrospena** [comment]
So, time for a call for proposals (CFP). I have repurposed the one from last year because I am lazy ;)

I am assuming we are sticking to the same format: 45-min talks with 15 mins for transitions. This is quite a lot of time, but to be honest, I found it nice last year. It makes for a relaxed devroom, where talks can run over a bit, and there is plenty of time for people to leave and come into the room. 

We have been moved to Saturday (January 30th), which is the big day :) That means we need to fill between 7.5 and 8.5 hours of content, beginning at 10.30 and running until between 18.00 and
19.00.

This is the CFP draft I've put together: please feel free to change it as you see fit. I'd like to send the CFP on Tuesday 27th October in the morning. So if you have any feedback, please speak by Monday end of day. Thanks!

# Open Source Design Devroom Call for Participation

This is the Call for Participation for the FOSDEM 2016 Open Source Design devroom. We invite you to send in your  submissions about design work on free, libre and open source projects.

FOSDEM (https://fosdem.org) is the Free and Open source Software Developers’ European Meeting, a free and non-commercial two-day weekend event that offers open source contributors a place to meet, share ideas and collaborate.

After a successful run in 2015, there will be again a design devroom at FOSDEM 2016: a full day of talks about design work on free, libre and open source projects. The Open Source Design devroom will take place on Saturday, January 30th.

We mean ‘design’ in the broadest sense: user research, interface and interaction design, visual design, typography and accessibility – all in the context of open source projects, which we believe introduces unique challenges. Designers and developers working with designers (or on a design-minded project) are welcome to submit proposals.

Possible topics include:

```
FOSS design tools
Design techniques and deliverables
Design - engineering collaboration approaches
User engagement and design research activities
Project case studies
```

# What and how to submit

RECORDING AND LICENSING:

The talks in the Open Source Design devroom will be audio and video recorded, and possibly streamed live too. By submitting a proposal you consent to be recorded and agree to license the content of your talk under a Creative Commons (CC-BY) license.

FORMAT:

45-minute presentations (it is up to you to leave time for questions, but we recommend you do so).

SUBMISSION DEADLINE:

1st December 2015

HOW TO SUBMIT:

All submissions are made in the Pentabarf event planning tool at

https://penta.fosdem.org/submission/FOSDEM16/

When submitting your talk in Pentabarf, make sure to select the ‘Open source design devroom’ as the ‘Track’.

If you already have a Pentabarf account from a previous year, please reuse it: create an account if, and only if, you don’t have one from a previous year. If you have any issues with Pentabarf, do not despair: contact belenbarrospena at gmail dot com

# Who coordinates this Open Source Design devroom thing

The Open Source Design Devroom is coordinated by the Open Source Design crowd: http://opensourcedesign.net/

**Comment by jancborchardt** [comment]
> I am assuming we are sticking to the same format: 45-min talks with 15 mins for transitions.

I would actually say we cut the talk time in half because 45 minutes I felt was too long. I liked this a lot at Libre Graphics Meeting where it ensured that more projects were presented and people got less bored.

20 minutes should absolutely suffice to present, and then we have 10 minutes between each talk for transitions and maybe questions. Although most questions are better to be talked about directly with the speaker as it respects everyone’s time and also already kickstarts a personal discussion between the people interested in the topic.

**Comment by belenbarrospena** [comment]
<blockquote><p>I am assuming we are sticking to the same format: 45-min talks with 15 mins for transitions.</p>
<p>I would actually say we cut the talk time in half because 45 minutes I felt was too long. I liked this a lot at Libre Graphics Meeting where it ensured that more projects were presented and people got less bored.
20 minutes should absolutely suffice to present, and then we have 10 minutes between each talk for transitions and maybe questions.</p></blockquote>

I am not against this, but I think we need to be clear on what this would mean:
1. A very different devroom to last year's: much more fast paced and more intense. 
2. Last year a few sessions exceeded their allocated time, but because we had plenty of time for the changes, there was no need to cut the speakers short. If we go for the 20-10 format, we will have to cut the speakers short and be quite tough with the timing.
3. Since we have to fill between 7.5 and 8.5 hours of content, we'll need between 15 and 17 sessions. Last year we received 17 submissions, 6 of them from people involved in coordination (nothing wrong with that by the way, but we might need to submit stuff ourselves this year as well to cover all the slots). I'd say we'll also need to do a better job in advertising the CFP than we did last year to get enough submissions. 
4. No chance to go in depth into any topic: 20 minutes is just enough to provide an introduction to something. Again, nothing wrong with that: just different kind of content.

I genuinely don't mind one way or another, but we need to go into the 20-10 format with our eyes wide open as to what we are getting ourselves into. 

It would be good to hear what others think, so please speak up! What do you prefer? 45-15 or 20-10 sessions?

**Comment by belenbarrospena** [comment]
asheesh in the irc channel said he also likes the shorter format (20-10). Just making a note of it for posterity :)

**Comment by belenbarrospena** [comment]
Since the people who spoke up preferred the 20 minute format, that's what we are going with.

I've submitted a pull request to the events repo to add the CFP
https://github.com/opensourcedesign/events/pull/23

Could someone review and merge if ok?

Thanks!

Belén

**Comment by jancborchardt** [comment]
Done, good stuff! :)

**Comment by belenbarrospena** [comment]
thanks @jancborchardt !

**Comment by belenbarrospena** [comment]
So the CFP is out
http://opensourcedesign.net/events/design/hack/meeting/2016/01/30/opensourcedesign-fosdem.html

Please spread the word, far and wide :)

**Comment by evalica** [comment]
Sorry for the late response. So any news on the design stand? Unfortunately I won't be able to stay full time there since I also need to participate in the Wikis stand. I could stay 2 hours there. Let me know if there are news. Ty

**Comment by ei8fdb** [comment]
I think unless we can get 4-6 people who can give 2 hours each it won’t be possible.

Can we have a show of hands? Say 10AM (move the times as you see fit) who can take slots:

10:00-11:00:
11:00-12:00:
12:00-13:00: Bernard
13:00-14:00: Bernard
14:00-15:00:
15:00-16:00:
16:00-17:00: Caty (evalica)
17:00-18:00: Caty (evalica)

Any takers?

**Comment by evalica** [comment]
I've edited your response with slots from 16-18.

**Comment by plastelina** [comment]
I can cover whole day, so there are always two people.

**Comment by ei8fdb** [comment]
Thanks @plastelina thats good news.

@belenbarrospena also had an idea: why not do the design desk on Sunday? That way the design track is finished and we'd all have more time...

Thoughts?

**Comment by jancborchardt** [comment]
Yeah, so the design desk should definitely not be on the same day as the room. I was under the impression that was the plan anyway? I think we only said Saturday for the design desk because last year the room was on Sunday?

**Comment by belenbarrospena** [comment]
This year the design devroom in on Saturday, so the design desk would be Sunday. I also think it's much better if we do it only the day we don't have devroom. We can publicise the activitity during the devroom on Saturday. If we go for Sunday, I should be able to put in 2-4 hours.

Boy, this is going to be loads of fun :)

**Comment by belenbarrospena** [comment]
Just FYI, we have been allocated room AW1.120. This might be the same room we had last year (at least is the same building). The room seems to fit 74 people (it's a small room). Nothing we can do about it really. It will be cozy ;)

**Comment by belenbarrospena** [comment]
Hi all,

Wondering if any of you would give me a hand spreading the word about the Open Source Design Devroom. Something like sending an email to the mailing lists / projects you are involved with, or using Twitter, Facebook, whatever you like :) 

So far, I've sent emails to the these mailing lists:

BCS-HCI (BCS-HCI@jiscmail.ac.uk)
LibreOffice design (design@global.libreoffice.org)
Firefox OS development (dev-fxos@lists.mozilla.org)
OpenITP UX (ux@lists.openitp.org)
London IA (london-ia@yahoogroups.com)

Thanks!!

Belén

**Comment by jancborchardt** [comment]
Also, all the folks from here should probably submit something! Calling @bnvk @plastelina @GarthDB @Xaviju @elioqoshi @evalica @danilapellicani @gillisig @simonv3 and everyone :)

**Comment by belenbarrospena** [comment]
Yes, that too! :) Thanks, @jancborchardt

**Comment by bnvk** [comment]
I don't have the bandwidth to put together a proposal and a talk.
But hopefully one of you fine folks will and then I can attend
and participate in the dev room :)

**Comment by elioqoshi** [comment]
Already proposed a talk :)

**Comment by GarthDB** [comment]
A little too close to my wife's due date to be travelling out of the country.

**Comment by belenbarrospena** [comment]
Congrats @GarthDB ! That's a fairly good excuse ;)

**Comment by GarthDB** [comment]
@belenbarrospena thanks! If they would just rearrange the whole event around my schedule that would be super convenient.

**Comment by Xaviju** [comment]
Hi! I've been extremely busy and I've unable to read tons of emails in my inbox (that's great!). I did not proposed a talk but I would like to share what I would love to hear: a talk about how an open source project manages open source design PRs and sometimes PRs that fixes issues but breaks design. 
- What is the workflow to keep it consistent?
- How they manage it? 
- How is the interaction with the user?
- How do you manage PRs from non designers that fixes issues, but breaks design?  
  I could propose a talk on how we manage it in Taiga, but i think I have more questions than answers...It would be nice to have open discussion talks :)

**Comment by belenbarrospena** [comment]
@Xaviju You are still on time to propose a talk, and I think you really should. I have my own way of dealing with all those things, but like you I have more questions than answers, and I suspect so do many others in this group. The point of these talks is sharing the little answers we have individually, so that they might answer the group's questions... or something along those lines but less confusing ;)

We would all love to hear how you manage the design work in Taiga, so head to Pentabarf and submit a talk! It doesn't need to be very elaborate: almost anything will do :)

Cheers

Belén

**Comment by jancborchardt** [comment]
Yeah @Xaviju @bnvk the talks are only 20 minutes, so it would be great that even if you can’t prepare a bunch of slides to just for example share how you do design work in your projects. :)

**Comment by elioqoshi** [comment]
I might submit another session. I did this screencast tutorial for SitePoint lately:
https://www.sitepoint.com/premium/screencasts/logo-thumb-rules

Would you think it makes sense to show it and I elaborate further on the techniques I've used? 
I only used open fonts in my logos

What do you think?

**Comment by ei8fdb** [comment]
Hi all,

I’ve already sent the CFP to the following lists:

Tor UX List ux@lists.torproject.org
The Guardian Project List guardian-dev@lists.mayfirst.org
Liberation Tech liberationtech@lists.stanford.edu
Gimp UX List gimp-gui-list@gnome.org
VLC Developers for Android android@videolan.org
GNOME Design community <sent to the design community people>
ADIUM <sent to the project leaders>
UK Government Design and research lists <unfortunately they’re closed lists :( >
Whisper Systems (Signal and TextSecure developers) whispersystems@lists.riseup.net

I’ll send it out on the Twitters too and bug people to retweet it too.

Bernard

**Comment by belenbarrospena** [comment]
@ei8fdb Thanks for that! 

@elioqoshi I would submit it. Last year there were lots of developers in the room. They might benefit from this type of content. And me, being an interaction designer and knowing pretty much nothing about logo design, would learn a lot too :)

**Comment by jancborchardt** [comment]
@elioqoshi yeah, you should totally give a talk about your logo design! :)

**Comment by danilapellicani** [comment]
dang the deadline is  tomorrow :o
I think I will attend but I have no time to prepare a talk :/

Danila Pellicani
D https://www.linkedin.com/in/danilapellicanio
https://www.linkedin.com/in/danilapellicani / Make
http://www.danilapellicani.it/ / Say https://twitter.com/danilapellicani

**Comment by belenbarrospena** [comment]
@danilapellicani: the deadline is not until the 1st, so you still have time! :) 

And in any case we'll have to extend it because we have only 4 proposal so far :/ I'd say we can push it until December 8th. If nobody is opposed, I will probably announce the extension tomorrow evening.

**Comment by jancborchardt** [comment]
Everyone who thinks they do not have time to prepare a talk now: submit a title only and add the abstract later. That's possible right @belenbarrospena?

Each person who is in this org and at FOSDEM should share their work in the devroom, don't be shy or think you have too little time. :)

**Comment by elioqoshi** [comment]
@jancborchardt how about if we create a panel to discuss about x topic? Several of us could join on stage for the conversation

**Comment by jancborchardt** [comment]
@elioqoshi yeah, that would be cool. We could do several short panel discussions, each 20 minutes or depending on how much gets filled with the talks we can even do the panels for 40 minutes. Maybe something like:
- Designers & developers: Finding the right communication channels and tone
- Barriers for designers getting involved in open source and what we can do to ease the process
- Barriers for developers getting designers involved in open source – and also how we can help
- …

What we should also definitely do again is reserve a slot (ideally in the middle of the day) for developers of projects who look for designers. They can quickly show their project and say what they need. Basically a live version of the Jobs board. We did that last year and it was quite successful. @belenbarrospena @ei8fdb

**Comment by elioqoshi** [comment]
Awesome! Let's make sure we get some chairs for the panel :P
Can we be a bit spontaneous about this or do we need to plan everything in beforehand?

**Comment by belenbarrospena** [comment]
these are all great ideas, but I think we need to plan, but maybe only a little bit ;)

If you could get these ideas submitted to Pentabarf, even if just with a 2-line description, that would be enough for us not to forget about them, make sure they are added to the program, and assign people to be responsible for them, so that they happen on the day. 

How does this sound?

and yes, as @jancborchardt says, feel free to submit something with just a title or a 2-line description, and we'll follow up as needed.

**Comment by jancborchardt** [comment]
@belenbarrospena @elioqoshi I’ll submit placeholder items for what I just mentioned, the panels and the live-job-board thingy. @elioqoshi feel free to submit more panel topics. :) We can then decide what we want to do when all is in.

**Comment by belenbarrospena** [comment]
Also, before I forget, I'll be sending a deadline extension announcement this evening / tomorrow morning. I think we should be able to extend until the 8th, which gives us 10 days to do the review and put the programme up in the FOSDEM website. 

If anybody has any concerns, please let me know before this evening.

**Comment by jancborchardt** [comment]
Should all be submitted from my side. @belenbarrospena was there an admin interface or such to see all the submissions to our devroom?

**Comment by belenbarrospena** [comment]
@jancborchardt 

https://penta.fosdem.org/report/review

If you have been given access. If you haven't let me know. I can ask the organisers to set you up as a devroom coordinator in Pentabarf. If you are subscribed to the devroom-managers mailing list, you can even ask them yourself! :)

**Comment by belenbarrospena** [comment]
@jancborchardt I can see your submissions in that report by the way.

**Comment by elioqoshi** [comment]
@belenbarrospena and co. Mozilla is sponsoring me, but they would like to ask if my sessions have been approved. Would it be possible to get a confirmation on this, or is it still too early?

**Comment by belenbarrospena** [comment]
@elioqoshi Submission deadline is end of day tomorrow and I am planning to send the sessions for review this week. Until then, I am not sure I can tell you for sure they are accepted (although there is a very good chance they will be). Can you wait? Or do they need an answer now?

**Comment by jancborchardt** [comment]
@belenbarrospena are we good with session proposals btw? Or should we do another round of advertising them?

And I assume the process will be the same for getting them approved, just by voting process again?

**Comment by belenbarrospena** [comment]
@jancborchardt We have 18 sessions submitted right now, which means we have enough content. But it can't harm to shout a bit if you can :) Deadline is tomorrow end of day.

Review can be done like last year. I will export all submissions to a spreadsheet, circulate it around, then anybody can review them and vote (yes / no) and we take the proposals with more votes. Does that sound ok?

**Comment by jancborchardt** [comment]
@belenbarrospena great stuff! I’m messaging a few more people as well.

**Comment by belenbarrospena** [comment]
@jancborchardt thanks!!

**Comment by belenbarrospena** [comment]
The CFP for the Open Source Design devroom closed on December 8th 2015. So it is review time!

Anybody can review the proposals (in fact, the more reviewers, the better). The process is very easy. This is what you need to know:

1) We can fit a maximum of 17 sessions (20-minute presentations with 10 minutes for transitions, starting at 10:30 and closing at 19:00).

2) All proposals are in this Google spreasheet

https://docs.google.com/spreadsheets/d/1bj906jUu57e5fy6TokLcGv_GUL6CmgRbe6SEsTipy7Y/edit?usp=sharing

3) Copy and paste the content of the spreadsheet into another spreadsheet. You can use whatever you want: Google Docs, Libre Office, even Excel ;) We are not fussy. You can use the File > Download as option to easily get a copy of the spreadsheet.

4) In the 'yes / no' column, write 'yes' for the sessions you'd like to have in the devroom. You have a maximum of 17 yeses (which means you need to leave 2 proposals out). If you have submitted something, please don't give a yes to your own session: that would not be cool ;) We will accept sessions based on the ratio of votes to yeses, instead of the total number of yeses.

5) Once you are done with your review, ping me by email, IRC or by leaving a comment in GitHub, and tell me how to lay my hands on your spreadsheet.

If you have any questions / comments, get in touch by email: belenbarrospena at gmail dot com.

The deadline for review is end of day (GMT) Sunday, December 13th. A preliminary schedule needs to be in Pentabarf by the 18th. 

Looking forward to getting your reviews.

Cheers.

**Comment by jancborchardt** [comment]
@belenbarrospena if we can fit 17 then it’s easy: I’ll just retract my proposals for »Panel: Barriers for designers getting involved in open source« and »Panel: Finding the right communication channels and tone between designers & developers« since these topics are already being tackled by other talks. They were mostly idea proposals anyway. :)

**Comment by evalica** [comment]
So we still vote or not? Another process of elimination would be to make sure the same presenter doesn't have more than 2 presentations for example (or ask if willing).

Regarding the voting process I guess we could have taken a column each, instead of copying and spreadsheet and send it offline. Depends if we want to have private votes or not. I don't have any problem having a transparent vote. 

Let me know if the proposals list if final (is especially important if we go with the copy spreadsheet approach). Thanks

**Comment by belenbarrospena** [comment]
Right: this is going to be a bit harder than last year :/

@evalica Yes, having a column per reviewer was another option. I didn't go for it because I believe reviews are better when blind (no author information) and anonymous (no reviewer information). I thought having to copy and paste / download the info in the spreadsheet would be a small price to pay in exchange for privacy and anonymity. I am sure some people would be ok with a non-anonymous vote (I would be too), but I figured just in case some people aren't, better to account for that.

I do agree about avoiding having speakers with more than one session. I submitted 2, but I rather not do the 2, to be honest. However, as you'll see below, I am not sure we have a choice. 

By the way, the list is final, unless someone comes out of the blue and wants to submit a new proposal. But we can deal with that on a case by case basis when (and if) it happens.

@jancborchardt and @ei8fdb have both proposed combining some of the sessions. These were mentioned:

1   Barriers for developers getting designers involved in their open source project
13  How can I contribute?
16  Panel: Barriers for designers getting involved in open source
17  Panel: Finding the right communication channels and tone between designers & developers

I think the above topics are important enough to be given more than 20 minutes. Maybe we could go for one hour (well, it will be 50 mins, since we need 10 mins for change). 

So is

10  Developers looking for designers? Show off your project!

Particularly if one or two people volunteer to get the jobs up on the site as people present, so that presenters come out of the session with their request published in the jobs board. I think we could block the last hour (18:00 to 19:00), and if we have no presentations, then we are done early and we wrap up and all go for a drink or two together :)

If we do the above, we still have slots for 13 20-minute sessions, but we only have 12 sessions. I don't think this is a problem: we can either allocate a bit more time to each session, block the first 30 minutes in the morning for introductions and announcements, or just finish at 18:30 instead of 19:00.

How does this sound? Any other suggestions?

**Comment by elioqoshi** [comment]
Love your idea Belen, if we can do that let's go with it!

**Comment by belenbarrospena** [comment]
Thanks for chipping in @elioqoshi 

I am just trying to get a bit more feedback from other folks who have been involved, and then I will start publishing the schedule and notifying the speakers.

By the way, the schedule can change, so we can still move things around if needed.

**Comment by evalica** [comment]
A. Regarding the topic of combining sessions: 1, 13, 16, 17, 10
- I really think that 1 and 16 are 'duplicates' so they should be definitely combined. 
- Regarding 17, the topic is a bit too focused on communication so this could be integrated in other presentation. Has similarities with 7.
- 13 should get a whole slot since knowing how to contribute, where and when IMO is very different than the involvement barriers. 13 is about the intention, 1+16+17 are about pre or post intention of contributing. 13 could be the first presentation in the bunch.
- Also I found 6, 7, 15 to be also in the same area, with minor nuances. So we will all talk about these problems and since the topics are recurrent is very clear that we all look for solutions :) An idea maybe would be to put all the mentioned presentations one after the other in order to reach some conclusions.
- 10 needs to be last and I hope we will have people interested in pitching their projects. If we will not have anyone coming we could maybe clean up the jobs board (from deprecated posts) or even give feedback or brainstorm together on a selected post (I would love to do that).

B. Regarding time of slots: either solution is fine with me (more time/slot or finish sooner). Still I would prefer not stalling time between sessions because some might get bored waiting for the next session, but we definitely need to respect the schedule for the people that will drop-in for a particular presentation.

Reference: 
1 Barriers for developers getting designers involved in their open source project
6 Designer's compromises in Open Source
7 Designers Vs developers 
10 Developers looking for designers? Show off your project!
13 How can I contribute?
15 Overcoming your designer ego
16 Panel: Barriers for designers getting involved in open source
17 Panel: Finding the right communication channels and tone between designers & developers

**Comment by elioqoshi** [comment]
@belenbarrospena and co.
It would be great if the talks could be approved by this week (or at least my talks) as I need to proceed with my sponsorship by Mozilla. It's getting a bit late and I need to get this going if I want to come to FOSDEM.

**Comment by belenbarrospena** [comment]
@evalica Thanks for the summary. It's really helpful!

@elioqoshi Yes, that's the plan. A provisional schedule needs to be up by Friday, so I am hoping to send speaker emails and have the schedule by the end of this week. I will send the emails first, since in your case is becoming urgent.

Cheers

**Comment by belenbarrospena** [comment]
Hello all,

Finally, I think I have a proposal for the devroom content. Hat tip to @evalica for her analysis of the submissions. The final list and order are very much in line with her thoughts. 

I have left the morning for projects, tools and techniques, and the afternoon for discussions about the relationship between design and development and how to get designers more involved.

This is not set in stone by any means, even if it goes into Pentabarf like this. So please, feel free to suggest changes. 

Here comes the tentative programme:
1. [10:30 - 10:50] Firefox OS: Why we exist- A brief look into the history and vision behind a web-based open-source operating system (Michael Henretty)
2. [11:00 - 11:20] Design for All versus Design for One and Adaptive User Interfaces (Christophe Strobbe)
3. [11:30 - 11:50] GPII - State of the art (Javier Hernández)
4. [12:00 - 12:20] Designing accessible applications (Samuel Thibault)
5. [12:30 - 12:50] Blender as virtual studio lighting playground - Using Blender's realtime rendering preview to explore lighting for photography or video setups (Tuomas Kuosmanen)
6. [13:00 - 13:20] Tips & Tricks for logo creation - Logo case studies from Mozilla & co. (Elio Qoshi)
7. [13:30 - 13:50] DIY mobile usability testing - A cheap, cheerful and portable lab for mobile testing (Michael Wood, Belen Barros Pena and Bernard Tyers)
8. [14:00 - 14:20] Open Source in non software design - Architectural, structural and other design domains could go open source (Michael Demetriou)
9. [14:30 - 14:50] Open source design in the UK Government? (Benard Tyers)
10. [15:00 - 15:20] Designing with and for developers - How UX landed in OpenEmbedded (Belen Barros Pena)
11. [15:30 - 15:50] Designers Vs developers - Can’t we all just get along? (Hollie Lubbock)
12. [16:00 - 16:20] Designer's compromises in Open Source (Ecaterina Moraru)
13. [16:30 - 16:50] Overcoming your designer ego - Experiences as a designer at Mozilla (Elio Qoshi)
14. [17:00 - 17:20] Connecting design students & open source projects - Why everyone benefits and how to do it (Jan-Christoph Borchardt)
15. [17:30 - 17:50] Getting designers involved in Open Source projects (Jan-Christoph Borchardt)
16. [18:00 - 18:20] How can I contribute? - Everything you wanted to know about contributing to open source but were afraid to ask. (Bernard Tyers)
17. [18:30 - 19:00] Developers looking for designers? Show off your project! - Open Source Design job board in person (Jan-Christoph Borchardt)

**Comment by elioqoshi** [comment]
Thank you Belen! That looks wonderful!

And sorry if I sounded a bit arrogant and pushing you. Just wanted to make sure I'd not be too late :)

**Comment by elioqoshi** [comment]
The only thing I might want to pick out (although it might not be a problem) is that @jancborchardt has 2 sessions in a row (17:00 and 17:30) which might be a bit too much to do at once. We should give him a little break maybe? :stuck_out_tongue:

**Comment by evalica** [comment]
Looks good - thank you @belenbarrospena for taking care of this. 
Just curios, do we still have a booth?

**Comment by belenbarrospena** [comment]
@elioqoshi yes, I saw that poor @jancborchardt ended up with 2 sessions back to back. We could switch "Connecting design students & open source projects" to the 15:30 slot. It's up to him, really. I'll wait to see what he says.

@evalica no, we don't. We decided in the end to camp on a table in the big cafeteria instead, to see what response we get. If people are enthusiastic about it, we could think of a booth next year. I believe @plastelina offered to organise things. Maybe she can tell us a bit about the plan.

**Comment by jancborchardt** [comment]
I’d say just cut the panel »Getting designers involved in Open Source projects« because it was only a panel proposal anyway. :) I don’t want to hog our room.

Also, the last session »Developers looking for designers? Show off your project!« doesn’t need my name since it’s a collaborative panel of just devs coming up and presenting their project.

**Comment by ei8fdb** [comment]
Hi all, 

Sorry been travelling and busy with work.

I had mentioned to @belenbarrospena about being happy to combine my proposal with @jancborchardt's. It's an important topic that needs to be discussed.

@jancborchardt your proposal of "Getting designers involved in Open Source projects" - I am very happy to have your panel "bleed" into the "How do I contribute" session.

Part of my proposal mentioned the possibility of a panel, or at least taking questions from the audience.

I'm very open to combining them into a session of two parts. 

I think it would be very good to hear from designers and researchers who are actually _doing it_, have experience, tales and suggestions on how to get involved, and then have that feed into the second "How do I contribute" session. My proposal was going to raise more questions that it'd answer!

Also in terms of having Jan having two sessions back-to-back, I'm happy to share the work in the panel session too.

What do others think? I just want to have the topic spoken about. I'm all ears.

Apart from that the schedule looks good. Well done @belenbarrospena.

**Comment by belenbarrospena** [comment]
Right folks: speaker emails are out, you should have yours :), and the schedule entered in Pentabarf. It will be uploaded to the website automatically at some point during the weekend. 

Any questions, changes, etc, let me know. 

Cheers

**Comment by plastelina** [comment]
Hi all, 
bad news on my side, well...good for me, not so much for FOSDEM - I've got a contract outside of Europe and I won't be able to make it to FOSDEM this year (which is quite devastating) . I'm still delivering projects as promised - logo, fonts for logo, print version of logo and stickers, will be ready with all of that by Tuesday- sorry had massive and a lot of deadlines running towards festive season. Where shall I sent the stickers to? Shall I just post them to FOSDEM location (the Brussels University and someone will pick them up or Delirium Cafe? - let me know and I will sort it out). Also, I'm happy to hold the video chat (skype / hangouts) for any advice on print / web / ui/ux projects. I can't promise I will be involved but I'm happy to point developers at people happy to help or resources that might help. 

I do apologies for not being as present as I would like to be in this community and popping in and out, but the nature of my work throws me at different direction / locations. 
Also, projects that I'm working on are quite time consuming hence me being a bit of a ghost here...there is so much we can do. 
Anyways, awesome job guys to organising all of that and wanting  / willing to introduce actual design in the process of future web / usability design. 

my skype in case we want to do life session of Q&A : karolina.ferenz ( happy to create an account just for that session and name it opensourcedesign) or hangouts karolina.ferenz@gmail.com, or slack

any questions: just ping me an email to: karolina.ferenz@gmail.com

Big up to all the people who put a MASSIVE effort into running this! (from humble designer perspective it means a lot!)

Be nice and work hard! Happy festive season to you all. speak soon 

@ei8fdb @belenbarrospena @elioqoshi @jancborchardt (the shouty one ;) - be nice and chill a bit :D )

**Comment by belenbarrospena** [comment]
No worries, @plastelina We all have lives to take care of :) Best of luck with that contract, and thanks for taking care of the stickers! That's awesome. 

I am not sure about sending them to Brussels: I don't know anybody there who could collect them, and I don't want to bother the organisers, since they probably have enough work already. I'd be happy to bring them with me: it's an easy train ride to Brussels from London. Anybody else a bit closer to @plastelina / Brussels would be willing to receive and bring the stickers? If nobody comes forth, I'll give you a London address to post them there. Would that work?

**Comment by belenbarrospena** [comment]
Hollie Lubock, one of the speakers in the devroom, has suggested going for dinner on Friday so that we can get to know each other. I think it's a nice enough idea :) Who would be interested in coming?

Also, any suggestions for places to go?

**Comment by ei8fdb** [comment]
Hi all,

Should we add the open source design dinner idea to the FOSDEM Fringe events website?

https://fosdem.org/2016/fringe/

**Comment by belenbarrospena** [comment]
@ei8fdb Looking at the events there, I am not sure a dinner fits in. Also, it seems to me that we kind of need to manage the numbers. We need to know more or less how many people are coming to the dinner in advance, so that Hollie Lubock, who very kindly agreed to organise it, can book the restaurant. Also, if we end up being 40 people, it will be hard to find a place. So, no?

**Comment by belenbarrospena** [comment]
Hi all,

I was wondering if there are any volunteers to help with the FOSDEM devroom logistics on Saturday morning. I have to set up the OpenEmbedded stand, so I am not sure how much time I am going to have to set up the devroom.

If I remember correctly, the thing involves mainly collecting a box from the organisers and setting up the video, which thankfully will be different than last year (hopefully easier). The organisers have sent this link around for the video stuff

https://github.com/FOSDEM/videobox

@jancborchardt, @yoroy and @ei8fdb helped last year. Would you guys be willing to take care of things again? Anybody else would be willing to help?

Thanks!

Belén

**Comment by ghost** [comment]
> @jancborchardt, @yoroy and @ei8fdb helped last year. Would you guys be 
> willing to take care of things again? Anybody else would be willing to 
> help?

Alas, I’m not going to make it to fosdem at all :-(

Schedule looks great, have a good one!

Roy

**Comment by jancborchardt** [comment]
:O @plastelina @yoroy we’ll miss you!

@belenbarrospena I’m down to help in the morning, I’m staying near the venue. @ei8fdb or anyone else, you too?

**Comment by belenbarrospena** [comment]
@yoroy Sorry to hear :(

@jancborchardt excellent. Thank you so much! 

If nobody else volunteers to help, I'll try to get out of the stand set up duties. You will need another pair of hands at least.

**Comment by jancborchardt** [comment]
Yeah, it would be really good if someone else could join in too, best 2 more people. Not sure yet when and how much I need to help setting up the ownCloud booth as well.

**Comment by ei8fdb** [comment]
I'll help out in the morning - I'll get the box of stuff from the organisers room. Help with the video streaming might be needed, so another pair of hands would be most welcome.

Of the others going, is anyone available?

**Comment by belenbarrospena** [comment]
@ei8fdb Thanks!

**Comment by belenbarrospena** [comment]
We are all done! The open source design devroom at FOSDEM 2016 was bigger, better and more fun than last year. A total success once more, showing the hunger for design in the FOSS community. 

Thanks to all the speakers and the people in this thread who made it happen. We'll be back next year.

**Comment by ei8fdb** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

\0/

Can we get slides from everyone's talks in the github repo so we can
make them public?

I've put mine here:

https://github.com/opensourcedesign/talks/tree/master/fosdem%202016

I'll document it a bit better when I get a chance...

Good idea?

On 31/01/2016 11:21, Belen Pena wrote:

> We are all done! The open source design devroom at FOSDEM 2016 was
> bigger, better and more fun than last year. A total success once
> more, showing the hunger for design in the FOSS community.
> 
> Thanks to all the speakers and the people in this thread who made
> it happen. We'll be back next year.
> 
> --- Reply to this email directly or view it on GitHub: 
> https://github.com/opensourcedesign/events/issues/22#issuecomment-1774
> 70809
> 
> -----BEGIN PGP SIGNATURE-----
> Comment: GPGTools - https://gpgtools.org

iQIcBAEBCgAGBQJWrfQ/AAoJENwn5lb+RG6iOh8P/21wJlM2cLsD++bA60smbfqF
V2a+V1I2PTne+eYn0l7JGrMIO7+GnL5MJ5vxN78FVvkkXzvlfpHSgsJhgJeNnjLQ
3b+Avd2fcnxKFp+EM8PHGPB9P+myFsnlf07d7isRFuT/8d4t0PNCK18dUuKuBTGg
vmLMZ1Lp2xImedEJdMB4paHvc79gbOW5hJZTqWf8aUBiWOF46QNkhDXpw1+8Kj9f
ncf9DBZ4AGtxCuv3Tz4j/gOt97cPvORceGxL03PJROxlX2buFSLzkG/O267spWqt
L20WjDMItQu+5U8nP6uoZr3uu2g2GdUu/7yDkWZ9ohk+uxYQ3l7e2Z9Y0qixSIpf
KWr9Z10OPnwslb9I6pE6/cUl22XSJV+LsBdosUxaBT59PIaqhSVCNMje5olUCVpq
RplfGcEyjqcqzufFjVZKPX7m8HjZZnPounknOz59s47bEtgJC4SSGliKd4+jGlfK
BgtS/zbFVnUMdK44UQTYlCgwyt2uTvG083nlGceM07GaQahh2yg0QFw0TVRmu+nr
Wwg0bbEqUwscEYRMBxklFwBO+YJ6/CzM48vjK2PTELYIYS2ohPDv23mlcv/UVAka
7y+7OCowX/AUAAq5TNGZW/W4W0dXSjHF7NNv3sZpi5zNE0auGuwZXx0A9SCkJ0uH
nlQVhno+XivRNe3ijjei
=svUX
-----END PGP SIGNATURE-----
